openpype/hosts/resolve/api/lib.py

```diff
@@ -192,48 +203,35 @@ def create_media_pool_item(fpath: str,
         object: resolve.MediaPoolItem
     """
     # get all variables
-    media_storage = get_media_storage()
     media_pool = get_current_project().GetMediaPool()
     root_bin = root or media_pool.GetRootFolder()
 
     # try to search in bin if the clip does not exist
-    existing_mpi = get_media_pool_item(fpath, root_bin)
+    existing_mpi = get_media_pool_item(files[0], root_bin)
```

Comment:
Would it be worth checking that it's not an empty list?
Also docstring for this arg is wrong.

Reply:
implemented via 61f381cb5cee14f9f2c85b6db04c9144f9818ac5